src/main/java/software/amazon/awssdk/crt/mqtt5/packets/PublishPacket.java

```diff
@@ -449,6 +449,30 @@ public PublishPacketBuilder withUserProperties(List<UserProperty> userProperties
          */
         public PublishPacketBuilder() {}
 
+        /**
+         * Creates a new PublishPacketBuilder with most common parameters set.
+         *
+         * @param topic The topic this message should be published to.
+         * @param payload The payload for the publish message.
+         */
+        public PublishPacketBuilder(String topic, byte[] payload) {
```

Comment:
Why would we want this one?  QoS is required, but payload isn't

Reply:
I encountered this case in tests, but failed to notice that that's a negative test. Removed this constructor